.vitepress/theme/styles/enhanced-tables.css

```
@@ -177,53 +177,53 @@
-  color: var(--vp-c-brand-1);
-  border-radius: 4px;
-  border: 1px solid var(--vp-c-brand-1);
-  border-opacity: 0.2;
-  white-space: nowrap;
+    padding: 0.125rem 0.375rem;
+    font-size: 0.875em;
+    background: var(--vp-c-brand-soft);
+    color: var(--vp-c-brand-1);
+    border-radius: 4px;
+    border: 1px solid var(--vp-c-brand-1);
+    border-opacity: 0.2;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)